SPDX-FileCopyrightText: 2021 fosslinux <fosslinux@aussies.space>

SPDX-License-Identifier: GPL-2.0-only

--- include/uapi/asm-generic/termios.h.bak	2021-07-23 14:23:51.330460544 +1000
+++ include/uapi/asm-generic/termios.h	2021-07-23 19:08:27.112810109 +1000
@@ -12,13 +12,6 @@
 #include <asm/termbits.h>
 #include <asm/ioctls.h>
 
-struct winsize {
-	unsigned short ws_row;
-	unsigned short ws_col;
-	unsigned short ws_xpixel;
-	unsigned short ws_ypixel;
-};
-
 #define NCC 8
 struct termio {
 	unsigned short c_iflag;		/* input mode flags */
